src/views/VerifyPrevAddress.vue

```diff
@@ -4,11 +4,12 @@
       <ProgressBar :nodeData="progressBarData"/>
       <h1>Verify you own {{alias}}@nav.community</h1>
       <div class="input-container">
-        <ListEntry><span slot="number" class="number">1</span><span slot="text" class="text small">Copy the message below</span></ListEntry>
+        <ListEntry><span slot="number" class="title number">1</span><span slot="text" class="text small">Copy the message below</span></ListEntry>
         <Copybox>signmessage {{aliasCurrentAddress}} {{alias}}@nav.community</Copybox>
         <br /> <br />
+        <br /> <br />
         <!-- <div><Button @click="copyText()">{{copied ? "Copied" : "Copy"}}</Button></div> -->
-        <ListEntry><span slot="number" class="number">2</span><span slot="text" class="text small">Now open your NavCoin wallet, open console through Help -> Debug and pastes the message, then hit Enter on your keyboard. An response message (verification code) will be created, copy and paste it here</span></ListEntry>
+        <ListEntry><span slot="number" class="title number">2</span><span slot="text" class="text small">Now open your NavCoin wallet, open console through Help -> Debug and pastes the message, then hit Enter on your keyboard. An response message (verification code) will be created, copy and paste it here</span></ListEntry>
         <TextInput v-model="addressVerification" :inputEvent="verifySignature">
```

Comment:
This input box acts differently from the one on the 'Verify Address' page (that you see when created a new alias).
It doesn't have the blue border when active.

Reply:
fixed